runtime/doc/Makefile

```diff
@@ -51,12 +30,6 @@ tags: doctags $(DOCS)
 doctags: doctags.c
 	$(CC) doctags.c -o doctags
 
-manpages: $(MANPAGES)
-
-# OSX groff doesn't support utf-8 as input encoding, so this won't work there.
-.1.man:
-	groff -k -mandoc -Tutf8 $< | sed -e s/.^H//g > $@
```

Comment:
After seeing this[1] I wasn't sure that it's actually true. Assuming a moderately recent (last decade) version of groff, man pages written in mdoc don't need any pre-processing; you just type `man my-program` and your man page is formatted on the fly, then displayed.

Could you test how `nvim.1` and `nvimtutor.1` look on OS X? You should be able to view them using `man -l`.

[1]: https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/groff.1.html